subworkflows/nf-core/vcf_validate_small_variants/main.nf

```diff
@@ -0,0 +1,185 @@
+include { RTGTOOLS_VCFEVAL      } from '../../../modules/nf-core/rtgtools/vcfeval/main'
+include { RTGTOOLS_ROCPLOT      } from '../../../modules/nf-core/rtgtools/rocplot/main'
+include { HAPPY_HAPPY           } from '../../../modules/nf-core/happy/happy/main'
+
+workflow VCF_VALIDATE_SMALL_VARIANTS {
+
+    take:
+    ch_vcf                          // [mandatory] channel: [ meta, vcf, tbi, truth_vcf, truth_tbi, regions_bed, targets_bed ]
+    ch_fasta                        // [happy only] channel: [ meta, fasta ]
+    ch_fasta_fai                    // [happy only] channel: [ meta, fasta_fai ]
+    ch_vcfeval_sdf                  // [vcfeval only] channel: [ meta, sdf ]
+    ch_happy_false_positive_regions // [optional] channel: [ meta, false_positives_bed ]
+    ch_happy_stratification_tsv     // [optional] channel: [ meta, stratification_tsv ]
+    ch_happy_stratification_beds    // [optional] channel: [ meta, [stratification_beds] ]
+    tools                           // [mandatory] value: A comma-delimited list of the tools to use for validation (happy,vcfeval)
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    happy_vcf = Channel.empty()
+    happy_tbi = Channel.empty()
+    happy_indel_roc = Channel.empty()
+    happy_indel_roc_pass = Channel.empty()
+    happy_snp_roc = Channel.empty()
+    happy_snp_roc_pass = Channel.empty()
+    happy_roc = Channel.empty()
+    happy_summary = Channel.empty()
+    happy_extended_csv = Channel.empty()
+
+    vcfeval_true_positive_vcf = Channel.empty()
+    vcfeval_true_positive_vcf_tbi = Channel.empty()
+    vcfeval_false_negative_vcf = Channel.empty()
+    vcfeval_false_negative_vcf_tbi = Channel.empty()
+    vcfeval_false_positive_vcf = Channel.empty()
+    vcfeval_false_positive_vcf_tbi = Channel.empty()
+    vcfeval_true_positive_baseline_vcf = Channel.empty()
+    vcfeval_true_positive_baseline_vcf_tbi = Channel.empty()
+    vcfeval_summary = Channel.empty()
+    vcfeval_phasing = Channel.empty()
+    vcfeval_snp_roc = Channel.empty()
+    vcfeval_non_snp_roc = Channel.empty()
+    vcfeval_weighted_roc = Channel.empty()
+
+    rtgtools_snp_png_rocplot = Channel.empty()
+    rtgtools_non_snp_png_rocplot = Channel.empty()
+    rtgtools_weighted_png_rocplot = Channel.empty()
+
+    rtgtools_snp_svg_rocplot = Channel.empty()
+    rtgtools_non_snp_svg_rocplot = Channel.empty()
+    rtgtools_weighted_svg_rocplot = Channel.empty()
+
+    list_tools = tools.tokenize(",")
+
+    if("happy" in list_tools){
```

Comment:
Maybe change this into a `switch` statement?

Reply:
Will this still work if both tools have to be run?

Reply:
Mainly just to give users more flexibility on what tools they want to run (will be especially useful if more tools were to be added later)